docs/static/monitoring/monitoring-mb.asciidoc

```diff
@@ -23,28 +25,28 @@ Want to use {agent} instead? Refer to <<monitoring-with-elastic-agent>>.
 ==== Disable default collection of {ls} monitoring metrics
 
 --
-The `monitoring` setting is in the {ls} configuration file (logstash.yml), but is
-commented out: 
+Set the `monitoring.enabled` to `false` in logstash.yml to disable to default monitoring:
```

Comment:
I have never seen `monitoring.enabled: false` param in the logstash.yaml and I believe [`xpack.monitoring.enabled`](https://github.com/elastic/logstash/blob/main/config/logstash.yml#L341) is different although their functionality crosses.

Reply:
If either `xpack.monitoring.enabled: false` or `monitoring.enabled: false` is explicitly set, then monitoring functionality will be disabled https://github.com/elastic/logstash/blob/efa83787a565064fbf2f4327741455f8863aefe2/x-pack/lib/monitoring/monitoring.rb#L160

Reply:
But, yes, at one point when direct shipping was implemented, there was a move to using `monitoring.*` instead of `xpack.monitoring.*`, and I expect this is a fall out from then